README.md

```diff
@@ -0,0 +1,74 @@
+# Document Merge Service
+
+[![Build Status](https://travis-ci.com/adfinis-sygroup/document-merge-service.svg?branch=master)](https://travis-ci.com/adfinis-sygroup/document-merge-service)
+[![Pyup](https://pyup.io/repos/github/adfinis-sygroup/document-merge-service/shield.svg)](https://pyup.io/account/repos/github/adfinis-sygroup/document-merge-service/)
+[![Black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/adfinis-sygroup/document-merge-service)
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
+
+Merge Document Template Service
```

Comment:
Maybe add a short introduction? This doesn't really explain anything more than the repo's name or the title above.
What use cases is this for? Who's the target audience?

Also, how to use (example API calls) would be very helpful as well

Reply:
Tried to clarify introduction sentence.

In terms of examples see Usage section just below.